daffodil-runtime1/src/main/scala/org/apache/daffodil/processors/parsers/SequenceParserBases.scala

```diff
@@ -268,24 +283,36 @@ abstract class SequenceParserBase(
               pstate.diagnostics = diagnosticsBeforeAttempt
             }
 
-            // We successfully parsed a discriminator, but failed to parse the discriminated content.
-            // Do not continue trying to parse other memebers of the unordered sequence.
+            // We successfully parsed a discriminator in an unordered sequence,
```

Comment:
I think you mean We successfully parsed a discriminator within a child of an unordered sequence, yes? 

So the resultOfTry value UnorderedSeqDiscriminatedFailure means that we failed a child, it was a child of an unordered sequence, and that failure occurred after the discriminator (for this child alternative) had evaluated to true.  Do I have that right?

Reply:
Yep, i'll update the comment.